### PR TITLE
Add User Interface section in UG

### LINKED ISSUE #268: [PE-D] Missing information in UG
UG does not explain the meanings of different colours for the properties 

-------------
Labels: `severity.Low` `type.DocumentationBug`
original: HuaiZe/ped#4

### PR DESCRIPTION
**Pull request by candyhy**
Defines Ui color scheme and property type icons

### CODE REVIEW

**Comment by candyhy** [comment]
Added image to define colour scheme of Ui
UiColorscheme

File: docs/UserGuide.md [hunk 1]
```diff
@@ -230,28 +247,36 @@ Description:
...
-### 3.6 Updating status
+### 4.6 Updating status
 
-#### 3.6.1 Updating the status of a property : `update`
+#### 4.6.1 Updating the status of a property : `update`
 
 Updates the status of a property from Option to Purchase, to Sales and Purchase Agreement to Completion
 
-Formats:
+Format:
```
Comment (dvdweien) [inline docs/UserGuide.md:279]: thanks man

File: docs/UserGuide.md [hunk 2]
```diff
@@ -530,7 +555,7 @@ Action | Format, Examples
 **Edit property** | `edit property INDEX [n/NAME] [t/PROPERTY_TYPE] [a/ADDRESS] [p/POSTAL_CODE] [d/DEADLINE] [r/REMARKS] [cn/CLIENT_NAME] [cc/CLIENT_CONTACT_NUMBER] [ce/CLIENT_EMAIL] [ca/CLIENT_ASKING_PRICE] [tags/TAGS_SEPARATED_BY_COMMAS]`<br><br> e.g.,`edit property 1 r/Urgent to sell cc/96011846`
 **Edit appointment** | `edit appointment INDEX [n/NAME] [r/REMARKS] [d/DATE] [t/TIME]`<br><br> e.g.,`edit appointment 3 d/21-07-2021 t/1500`
 **Remove an entry** | `delete appointment INDEX` <br> e.g. `delete appointment 7` <br><br> `delete property INDEX` <br> e.g. `delete property 7`
-**List all** | `list all`
+**List all property and appointment** | `list all`
```
Comment (dvdweien) [inline docs/UserGuide.md]: should it be properties and appointments instead?

**Review by dvdweien — COMMENTED** [review]
Just a minor suggestion

File: docs/UserGuide.md (hunk 2 above)
Comment (candyhy) [reply docs/UserGuide.md]: oh I wanted to keep it consistent but sure ill update it

**Review by dvdweien — APPROVED** [review]
LGTM

**Pull request merged**